prometheus/prometheus.yml

```diff
@@ -32,7 +32,7 @@ scrape_configs:
     scrape_interval: 5s
     dns_sd_configs:
       - names: ['tasks.gateway']
-        port: 8080
+        port: 8082
```

Comment:
From an operational standpoint I can't merge this PR until we do a new release of the gateway, this needs to be two separate PRs.

Reply:
Okay .. so these prometheus configs should be put in a different PR and one PR to add the gateway changes ?

Reply:
Yes